KIPs/kip-34.md

```diff
@@ -325,8 +325,8 @@ Each keyring class uses the `PrivateKey` class, which has one private key as a m
 
 | Method | Description |
 | ----------- | ----------- |
-| sign(txHash: String, chainId: String): SignatureData | Signs the transaction based on the transaction hash and chain id and returns the signature. |
-| sign(txHash: String, chainId: int): SignatureData | Signs the transaction based on the transaction hash and chain id and returns the signature. |
+| sign(txSigHash: String, chainId: String): SignatureData | Signs the transaction based on the transaction hash and chain id and returns the signature. |
```

Comment:
@jimni1222 It would be better to introduce where to find sigHash (e.g., https://docs.klaytn.com/klaytn/design/transactions/fee-delegation#rlp-encoding-for-signature-of-the-sender-1) as a link.

Reply:
I applied your review. Please review again :)

Reply:
Thanks for the quick response!